docs/root/configuration/listeners/network_filters/zookeeper_proxy_filter.rst

```diff
@@ -25,17 +25,38 @@ in the configuration snippet below:
 
 .. code-block:: yaml
 
-  filter_chains:
-  - filters:
-    - name: envoy.filters.network.zookeeper_proxy
-      typed_config:
-        "@type": type.googleapis.com/envoy.extensions.filters.network.zookeeper_proxy.v3.ZooKeeperProxy
-        stat_prefix: zookeeper
-    - name: envoy.filters.network.tcp_proxy
-      typed_config:
-        "@type": type.googleapis.com/envoy.extensions.filters.network.tcp_proxy.v3.TcpProxy
-        stat_prefix: tcp
-        cluster: ...
+  listeners:
+    - name: main
+      address:
+        socket_address:
+          address: 127.0.0.1  # Host that zookeeper clients should connect to.
+          port_value: 12181   # Port that zookeeper clients should connect to.
+      filter_chains:
+      - filters:
+        - name: envoy.filters.network.zookeeper_proxy
+          typed_config:
+            "@type": type.googleapis.com/envoy.extensions.filters.network.zookeeper_proxy.v3.ZooKeeperProxy
+            stat_prefix: zookeeper
+        - name: envoy.filters.network.tcp_proxy
+          typed_config:
+            "@type": type.googleapis.com/envoy.extensions.filters.network.tcp_proxy.v3.TcpProxy
+            stat_prefix: tcp
+            cluster: local_zk_server
+  clusters:
+    - name: local_zk_server
+      connect_timeout: 120s
+      type: LOGICAL_DNS
+      lb_policy: ROUND_ROBIN
+      http2_protocol_options: {}
```

Comment:
this will need to be updated to the newer http2 spec

you can use `git grep http2_proto | grep yaml`  to find other examples